internal/drivers/eks_with_eksctl/driver.go

```diff
@@ -157,15 +157,17 @@ func NewDriver(name string, opts Options) (drivers.Tester, error) {
 func (k *driver) eksctl(ctx context.Context, args ...string) error {
 	args = append(args, []string{
 		"--color", "false", // Disable color output
+		"--dumpLogs",     // Enable CloudFormation log dumping on failures
```

Comment:
These logs will be dumped to disk so they won't be in the error output below. Is that desired?